src/DetailsView/Styles/detailsview.scss

```diff
@@ -407,55 +387,12 @@ div.insights-file-issue-details-dialog-container {
             overflow: hidden;
             padding-left: calc(0.5vw + 13px); // 13px is Chevron width.
         }
-        .left-nav {
-            max-height: calc(100vh - (#{$detailsViewTotalHeaderHeight}));
-            height: calc(100vh - (#{$detailsViewTotalHeaderHeight}));
-            overflow-y: auto;
-            border-right: 1px solid $neutral-8;
-            .dark-gray {
-                color: $neutral-60;
-            }
-            .is-selected .button-flex-container {
-                color: $always-black;
-            }
-            .button-flex-container {
-                padding-left: 5%;
-                padding-right: 8%;
-                width: $detailsViewLeftNavWidth;
-                color: $primary-text;
-                .status-icon {
-                    font-size: 24px;
-                }
-            }
-            .ms-Nav-link {
-                height: fit-content;
-            }
-            .overview-label {
-                padding-left: 10%;
-                text-align: left;
-                margin-top: 6px;
-                margin-bottom: 6px;
-                display: flex;
-                flex-direction: column;
-            }
-            .overview-name {
-                line-height: 20px;
-            }
-            .overview-percent {
-                color: $neutral-55;
-                line-height: 16px;
-                font-size: 12px;
-            }
-        }
         .test-step-nav {
             .ms-Nav-compositeLink {
                 .button-flex-container {
                     width: 100%;
                 }
             }
-        }
-        .left-nav,
-        .test-step-nav {
             z-index: 100;
             background-color: $neutral-0;
             .index-circle {
```

Comment:
Think you can get rid of these styles. test-step-nav doesn't have anything that has 'index-circle' class.

Reply:
Also applies to line 489

Reply:
And 504